ovos-i2csound

```diff
@@ -98,6 +98,10 @@ main() {
     if [[ ${detection_results[MARK1]} == true ]] ; then
         echo "Installing and configuring WM8731 based sound HAT"
         dtoverlay proto-codec
+        dtoverlay disable-bt
+        dtoverlay miniuart-bt
+        dtparam audio=off
```

Comment:
In my oppinion this is not needed if we add this file;
https://github.com/OpenVoiceOS/ovos-buildroot/blob/develop/buildroot-external/board/ovos/raspberrypi/rootfs-overlay/etc/modprobe.d/bcm2835-alsa.conf

That way the HAT's found will always be preferred over the onboard auto selecting it as default because of it. However this keeps the other sound options available for when you want to switch outputs.

Reply:
the reasoning was just that the audio jack isnt exposed in a mk1, so no point in having it enabled

Reply:
Understand, but the HDMI is. On the other hand, running the GUI on the Mark1 might be just a bit to much anyway. However you could connect the HDMI to your nice receiver though